### ISSUE TITLE
2019.2.1 - pip.installed state fails with pip versions between 10.0 and 18.0

### ISSUE THREAD

**Issue by OrlandoArcapix** [saltstack/salt#54773]
### Description of Issue

The import of pip.exceptions fails, causing the pip_state module load to fail, with pip versions 10.0 thru 18.0.

### Setup

Install pip 18.0.

Create a pip.installed state.

### Steps to Reproduce Issue
Apply pip.installed state.

### Versions Report

```Salt Version:
           Salt: 2019.2.1

Dependency Versions:
           cffi: 1.12.3
       cherrypy: unknown
       dateutil: 2.7.2
      docker-py: 3.5.0
          gitdb: 2.0.5
      gitpython: 2.1.11
          ioflo: Not Installed
         Jinja2: 2.9.6
        libgit2: Not Installed
        libnacl: Not Installed
       M2Crypto: 0.31.0
           Mako: 1.1.0
   msgpack-pure: Not Installed
 msgpack-python: 0.5.6
   mysql-python: 1.2.5
      pycparser: 2.19
       pycrypto: 2.6.1
   pycryptodome: 3.7.3
         pygit2: Not Installed
         Python: 2.7.5 (default, Apr  9 2019, 14:30:50)
   python-gnupg: Not Installed
         PyYAML: 3.11
          PyZMQ: 15.3.0
           RAET: Not Installed
          smmap: 2.0.5
        timelib: Not Installed
        Tornado: 4.2.1
            ZMQ: 4.1.4

System Versions:
           dist: centos 7.6.1810 Core
         locale: UTF-8
        machine: x86_64
        release: 3.10.0-957.12.2.el7.x86_64
         system: Linux
        version: CentOS Linux 7.6.1810 Core```

**Comment by OrlandoArcapix** [comment]
PR https://github.com/saltstack/salt/pull/54772 submitted to resolve this issue

**Comment by OrlandoArcapix** [comment]
See also https://github.com/saltstack/salt/issues/54755

**Comment by smitelli** [comment]
If anyone's in a rush and needs a workaround _right now_, you can try installing the Python 2 version of pip 18.1 (and, presumably, any future version) via your system's package manager and re-run the `salt*` command that failed.

**Comment by dwoz** [comment]
Duplicate of #54762, resolved by #54807. This will be fixed in 2019.2.2
